### PR TITLE
Add property tests for the other mount types

### PR DESCRIPTION
**Pull request by garyb**
Also switched to using `-generics-rep` for deriving `Show`, as using the json codecs to show was sometimes hiding the problems that arose in codec failures.

### CODE REVIEW

**Comment by garyb** [comment]
Also switched to QC, as it's significantly faster (like it takes no time, vs several seconds).

I won't merge this until the `-strings` dependency thing is resolved.

File: src/Quasar/Mount/Couchbase/Gen.purs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{-
+Copyright 2016 SlamData, Inc.
```
Comment (natefaubion) [inline src/Quasar/Mount/Couchbase/Gen.purs]: Did you write this last year? 😛

Comment (garyb) [reply src/Quasar/Mount/Couchbase/Gen.purs]: Duh, I write all my code in the past.

**Review by natefaubion — APPROVED** [review]

**Comment by Cmdv** [comment]
I saw 47 files changed thought he really is a 🐇  but it was because he must of been watching too much back to the future

**Pull request merged**